fluentui_peoplepicker/src/main/java/com/microsoft/fluentui/peoplepicker/PeoplePickerTextView.kt

```diff
@@ -612,18 +632,20 @@ internal class PeoplePickerTextView : TokenCompleteTextView<IPersona> {
 
     private class TokenListener(val view: PeoplePickerTextView) : TokenCompleteTextView.TokenListener<IPersona> {
         override fun onTokenAdded(token: IPersona) {
-            if (view.shouldAnnouncePersonaAddition)
+            if (view.shouldAnnouncePersonaAdditionMap[token] != false)
                 view.tokenListener?.onTokenAdded(token)
             if (view.isFocused)
                 view.announcePersonaAdded(token)
             view.sendAccessibilityEvent(AccessibilityEvent.TYPE_VIEW_ACCESSIBILITY_FOCUS_CLEARED)
+            view.shouldAnnouncePersonaAdditionMap.remove(token)
         }
 
         override fun onTokenRemoved(token: IPersona) {
-            if (view.shouldAnnouncePersonaRemoval)
+            if (view.shouldAnnouncePersonaRemovalMap[token] != false)
```

Comment:
cant we directly do view.shouldAnnouncePersonaRemovalMap[token] == true
otherwise it will work for null also

Reply:
By default we will be announcing the persona removal and addition. If in any case we don't need the notification we have to set it to false in the corresponding map.